src/main/java/com/openmrs/migrator/core/services/impl/PDIServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.openmrs.migrator.core.services.impl;
+
+import com.openmrs.migrator.core.services.PDIService;
+import java.io.InputStream;
+import org.pentaho.di.core.KettleEnvironment;
+import org.pentaho.di.core.Result;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.logging.LogLevel;
+import org.pentaho.di.job.Job;
+import org.pentaho.di.job.JobMeta;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PDIServiceImpl implements PDIService {
+
+  private static Logger LOG = LoggerFactory.getLogger(PDIServiceImpl.class);
+
+  @Override
+  public boolean runJob(InputStream xmlStream) throws KettleException {
```

Comment:
@cfaife sorry I missing this one earlier when I reviewed: pls rename this parameter: `xmlStream` to something more descriptive of what it actually represents, i.e. something like `pdiJobFileStream`

Reply:
done